src/transform/ancestor.js

```diff
@@ -118,7 +118,7 @@ function canBeLifted(doc, range) {
   }
 }
 
-// :: (Node, Pos, ?Pos) → bool
+// :: (Node, Pos, ?Pos) → ?{found: any, range: any}
```

Comment:
I would consider that cleaner in this case. As a general rule you shouldn't modify your code-base to make Flow happy, though, since Flow is just not powerful enough yet. Also, your code is not strictly typed and hasn't been written to be strictly-typed. So, if you would do this because you consider it better, then do it, but if you would just do it to make Flow happy, then don't. You won't be able to make Flow happy without hacks anyway.